README.md

```diff
@@ -50,7 +50,7 @@ crowd.dev is open-source, built with developers in mind, available for both host
 - You will never be locked in by us. Our interests as a company are aligned with you and we need to make sure that we always deliver enough value to you with our commercial offering in relation to our pricing.
 
 **To our developer community:**
-- You can self-host crowd.dev to centralize data for your community or company while keeping full control over your data.
+- You can actively engage, contribute to our roadmap, and shape crowd.dev into the tool you've always desired.
```

Comment:
Hey! Your suggestion is nice! But simultaneously, it also completely changes the initial sentence's meaning. Could you revert this line?

Reply:
Sure!

Reply:
@joanagmaia check updated README.md